locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -3004,6 +3006,14 @@ protected static double toDouble(Object v) {
         }
     }
 
+    protected static int[] toIntPortArray(Object v) {
+        Collection<String> inboundPortEntries = toListOfStrings(v);
```

Comment:
Thanks, @neykov.  I will add support for the `[1, 2, 3]` format and Java int arrays.  I'll also add test(s) in `JcloudsLocationTest`.